test/controllers/questions_controller_test.rb

```diff
@@ -9,6 +9,11 @@ class QuestionsControllerTest < ActionDispatch::IntegrationTest
     @token = @authentication.token
   end
 
```

Comment:
This may also require dev changes, but I believe we should also have a case such that admins are unable to create a question if the form is empty

Reply:
Agreed! Let's make a new story for that or include it in the better error handling.

Reply:
I'll make another story for it 👍